tests/pruners_tests/test_patient.py

```diff
@@ -0,0 +1,81 @@
+from typing import List
+
+import pytest
+
+import optuna
+
+
```

Comment:
How about testing that `PatientPruner` raises `ExperimentalWarning`?

Reply:
Great!  78e5672